src/media/js/main.js

```diff
@@ -84,14 +84,19 @@ require(
         // Do some last minute template compilation.
         logger.log('Reloading chrome');
 
+        if ((!settings.mktNavEnabled || !helpersLocal.isDesktop()) &&
```

Comment:
Been thinking about moving the "last-minute template rendering" logic into header_footer.js. It could be renamed, but it would handle all of the header, footer, and banner rendering.